### ISSUE TITLE
Add a method for Spring Boot to configure the new analytics feature

### ISSUE THREAD

**Issue by freddiN** [liquibase/liquibase#6501]
### Search first

- [x] I searched and no similar issues were found

### Description

I would like to see an easily usable configuration to configure the new analytics feature through Spring Boot.
So far its configurable through the maven client, the config file (liquibase.properties) and via env, but its not configurable through Spring Boot (setting a global env aside).
This would enable the spring boot guys to introrudce e.g. a spring.liquibase.xyz flag.

### Steps To Reproduce

Use 4.30.0 through Spring Boot

### Expected/Desired Behavior

Having a flag you can set to true or false to switch it on or off

### Liquibase Version

4.30.0

### Liquibase Integration

spring boot

### Are you willing to submit a PR?

- [ ] I'm willing to submit a PR (Thank you!)

**Comment by lalo-mx** [comment]
Workaround:
`System.setProperty("liquibase.analytics.enabled", "false");`

**Comment by rdhzl** [comment]
It's generally disappointing that analytics was made opt-out by default for the OSS Liquibase community. I hope there will be reconsideration about this.

**Comment by lbadam** [comment]
@freddiN thanks for filing this issue. It is something we will take into consideration. With the various workarounds already available it is a lower priority for us right now. To get it in faster, we would accept a PR from the larger community.

**Comment by lbadam** [comment]
@rdhzl I wrote extensively [in this PR](https://github.com/liquibase/liquibase/pull/6412) about our reasoning and why these anonymous analytics are helpful to the project overall. We have provided many ways to opt-out, and plan to continue with the current direction.

**Comment by filipelautert** [comment]
@freddiN A flag would be really nice! But even if we open the request now it will be present only in the next springboot version, so meanwhile workarounds here are still required.

**Comment by filipelautert** [comment]
> I still think its a good idea to have all relevant configuration at one place for Spring Boot/Liquibase users, so to me enabling them to introduce another spring.liquibase.xyz property for their users to use (even if its not now, but in the forseeable future) would be a benefit.

@freddiN sorry, I didn't mean that we should not add the flag. In fact it would be good to open the PR for **springboot** as soon as possible so it makes it to `3.5.0` , meanwhile  we can coordinate in Liquibase side to support it in `SpringLiquibase` class for 4.30.x+ . But until there using the System property is the way to go and it's "documented" here ;)

**Comment by lbadam** [comment]
I also agree with @filipelautert. Adding a reminder that analytics are also only included in 4.30.x+ and are not present in any previous versions.

**Comment by filipelautert** [comment]
https://github.com/liquibase/liquibase/pull/6516

**Comment by filipelautert** [comment]
Fixed in Liquibase side by https://github.com/liquibase/liquibase/pull/6516 . Now taking next steps in Springboot side.
